tests/reducers/medication-reducers.test.js

```diff
@@ -161,6 +161,35 @@ describe('Medication Reducers', () => {
     });
   });
 
+  describe('UPDATE_FHIR_MEDICATION_ORDER', () => {
```

Comment:
Note that you can place multiple "it" statements under a single "describe" to group all tests for that item